src/Vegetation/PlantHydraulics.jl

```diff
@@ -195,41 +199,46 @@ in section 6.4 "Bulk plant hydraulics model".
 function flux(
     z1,
     z2,
-    p1,
-    p2,
-    vg_α,
-    vg_n,
-    vg_m,
-    ν,
-    S_s,
-    K_sat1,
-    K_sat2,
+    ψ1,
+    ψ2,
+    plant_vg_α,
+    plant_vg_n,
+    plant_vg_m,
+    plant_ν,
+    plant_S_s,
+    plant_K_sat1,
+    plant_K_sat2,
 ) where {FT}
 
-    S_l1 = inverse_water_retention_curve(vg_α, vg_n, vg_m, p1, ν, S_s)
-    S_l2 = inverse_water_retention_curve(vg_α, vg_n, vg_m, p2, ν, S_s)
+    S_l1 = inverse_water_retention_curve(plant_vg_α, plant_vg_n, plant_vg_m, ψ1, plant_ν, plant_S_s)
+    S_l2 = inverse_water_retention_curve(plant_vg_α, plant_vg_n, plant_vg_m, ψ2, plant_ν, plant_S_s)
 
-    K1_p1 = hydraulic_conductivity(K_sat1, vg_m, S_l1)
-    K2_p2 = hydraulic_conductivity(K_sat2, vg_m, S_l2)
+    K1_ψ1 = hydraulic_conductivity(plant_K_sat1, plant_vg_m, S_l1)
+    K2_ψ2 = hydraulic_conductivity(plant_K_sat2, plant_vg_m, S_l2)
 
-    flux = -(K1_p1 + K2_p2) / 2 * ((p2 - p1) / (z2 - z1) + 1)
+    flux = -(K1_ψ1 + K2_ψ2) / 2 * ((ψ2 - ψ1) / (z2 - z1))
```

Comment:
What about the +1? is Psi = matric potential + z, or just matric potential?

Reply:
This was a mistake - forgot to put it back in after having removed it when thinking about the conversion. Psi is matric potential only